content/guides/references/configuration.md

```diff
@@ -800,17 +802,19 @@ and will not work:
 
 :::
 
-Solution: place the `baseUrl` property at the top level, outside the `env`
-object.
+Solution: place the `baseUrl` property outside the `env` object and inside a
```

Comment:
Instead of "inside a test type-specific" please say "inside the `e2e` testing-type specific object" and link to the `#e2e` section of this page. Also, the previous paragraph says that `baseUrl` is a "top-level config option" which is not correct. Please adjust that text to be accurate as well.